addons/l10n_bh/data/tax_report_full.xml

```diff
@@ -0,0 +1,399 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo auto_sequence="1">
+    <record id="l10n_bh_tax_report_full" model="account.report">
+        <field name="name">Full VAT Return</field>
+        <field name="root_report_id" ref="account.generic_tax_report"/>
+        <field name="country_id" ref="base.bh"/>
+        <field name="filter_fiscal_position" eval="True"/>
+        <field name="availability_condition">country</field>
+        <field name="column_ids">
+            <record id="l10n_bh_tax_report_full_base" model="account.report.column">
+                <field name="name">Base</field>
+                <field name="expression_label">base</field>
+            </record>
+            <record id="l10n_bh_tax_report_full_adjustment" model="account.report.column">
+                <field name="name">Adjustment/Apportionment</field>
+                <field name="expression_label">adjustment</field>
+            </record>
+            <record id="l10n_bh_tax_report_full_tax" model="account.report.column">
+                <field name="name">Tax</field>
+                <field name="expression_label">tax</field>
+            </record>
+        </field>
+        <field name="line_ids">
+            <record id="l10n_bh_tax_report_full_vat_sale" model="account.report.line">
+                <field name="name">VAT on sales</field>
+                <field name="hierarchy_level">0</field>
+                <field name="children_ids">
+                    <record id="l10n_bh_tax_report_full_1a" model="account.report.line">
+                        <field name="name">1(a). Standard rated sales at 10%</field>
+                        <field name="code">bh_1a</field>
+                        <field name="expression_ids">
+                            <record id="l10n_bh_tax_report_full_1a_base" model="account.report.expression">
+                                <field name="label">base</field>
+                                <field name="engine">tax_tags</field>
+                                <field name="formula">1(a) B</field>
```

Comment:
I do agree, I think it would be clearer with things like 1a_B or other things like this.
Now it won't break, but it feels a bit risky. I won't block it tho if we really think that's what we should have